src/Templates/src/templates/maui-blazor/MauiApp1/MacCatalyst/Info.plist

```diff
@@ -28,7 +28,5 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
-	<key>XSAppIconAssets</key>
-	<string>Assets.xcassets/appicon.appiconset</string>
```

Comment:
This looks good, also need to make the same change in https://github.com/dotnet/maui/blob/main/src/Templates/src/templates/maui-mobile/MauiApp1/MacCatalyst/Info.plist